src/components/Trigger/Trigger.tsx

```diff
@@ -72,7 +72,7 @@ export const Trigger: FC<TriggerProps> = ({
                         ])}
                         onClick={() => !!onClear && onClear()}
                     >
-                        <IconReject size={IconSize.Size12} />
+                        <IconTrash size={IconSize.Size12} />
```

Comment:
Agreed, I have added onDelete and showDelete props. It also still has the option to show a 'clear' icon.

Reply:
I guess @triggertoo meant to have `showDelete` instead of `clearable` and `onDelete` instead of `onClear`.

Also I don't understand the functionality as it is right now. What's the purpose to delete the whole input? I think the user should only be able to delete/clear an already selected color.